src/Nfts/Nfts.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import {storiesOf} from '@storybook/react-native'
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import {QueryClient, QueryClientProvider} from 'react-query'
+import {Provider} from 'react-redux'
+
+import {mocks} from '../../storybook'
+import getConfiguredStore from '../legacy/configureStore'
+import {SelectedWalletProvider} from '../SelectedWallet'
+import {YoroiWallet} from '../yoroi-wallets'
+import {StakePoolInfosAndHistories} from '../yoroi-wallets/types'
+import {Nfts} from './Nfts'
+
+const mockedAccountState = {
+  isDelegating: false,
+  isFetching: false,
+  lastFetchingError: '',
+  poolOperator: null,
+  totalDelegated: new BigNumber(0),
+  value: new BigNumber(0),
+}
+
+storiesOf('Nfts', module)
+  .add('Loading', () => {
+    const store = getConfiguredStore(true, true, {
```

Comment:
if you arent storing/reading anything from redux, there is no need to include it